gatsby/src/pages/schedule.js

```diff
@@ -35,6 +35,10 @@ const PageStyles = styled.div`
   .sz-session--service {
     background-color: #b4d7ff;
   }
+  .sz-modal-container .sz-session__title,
```

Comment:
This part looks good, but up above I see an error we might want to deal with.

Reply:
Line 33 has a double # tag

  .sz-session__card {
    background-color: ##5ba4f3 !important;
  }
  
  If we fix this, we might not even need the !important.